packages/tokenbank_v1/contracts/lib/SignatureUtil.sol

```diff
@@ -0,0 +1,52 @@
+/*
+
+  Copyright 2017 Loopring Project Ltd (Loopring Foundation).
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+pragma solidity ^0.5.11;
+
+
+/// @title Claimable
+/// @author Brecht Devos - <brecht@loopring.org>
+/// @dev Extension for the Ownable contract, where the ownership needs
+///      to be claimed. This allows the new owner to accept the transfer.
+contract SignatureUtil
```

Comment:
We can also just use the Open Zeppelin contract here:
- https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/cryptography/ECDSA.sol

Reply:
Oh, I see it doesn't have the `index` functionality that is used here. Then I guess not.

Reply:
Made it a library.